src/assets/locales/fr.json

```diff
@@ -0,0 +1,567 @@
+  "sign-in-modal.newsletter-signup-link": "S'inscrire pour les mises à jour",
+  "sign-in-modal.newsletter-signup-question": "Voulez vous que les news de Hubs soient envoyées dans votre boîte de réception?",
+  "sign-in-modal.prompt": "Connectez vous s'il vous plait",
+  "sign-in-modal.signin-message.add-owner": "Vous devrez vous connecter pour assigner des modérateurs.",
+  "sign-in-modal.signin-message.change-scene": "Vous devrez vous connecter pour changer la scène.",
+  "sign-in-modal.signin-message.close-room": "Vous devrez vous connecter pour fermer la salle.",
+  "sign-in-modal.signin-message.create-avatar": "Vous devrez vous connecter pour créer des avatars.",
+  "sign-in-modal.signin-message.favorite-room": "Vous devrez vous connecter pour ajouter cette salle à vos favorites.",
+  "sign-in-modal.signin-message.favorite-rooms": "Vous devrez vous connecter pour ajouter des rooms favorites.",
+  "sign-in-modal.signin-message.kick-user": "Vous devrez vous connecter pour éjecter les autres utilisateurs.",
+  "sign-in-modal.signin-message.mute-user": "Vous devrez vous connecter pour couper le micro des autres utilisateurs.",
+  "sign-in-modal.signin-message.pin": "Vous devrez vous connecter pour épingler des objets.",
+  "sign-in-modal.signin-message.remix-avatar": "Vous devrez vous connecter pour remixer des avatars.",
+  "sign-in-modal.signin-message.remix-scene": "Vous devrez vous connecter pour remixer des scenes.",
+  "sign-in-modal.signin-message.remove-owner": "Vous devrez vous connecter pour assigner des modérateurs.",
+  "sign-in-modal.signin-message.room-settings": "Vous devrez vous connecter pour changer les réglages de la salle.",
+  "sign-in-modal.signin-message.tweet": "Vous devrez vous connecter pour envoyer des tweets.",
+  "sign-in-modal.signin-message.unpin": "Vous devrez vous connecter pour détacher des objets.",
+  "sign-in-modal.title": "Se connecter",
+  "sign-in-modal.wait-for-verification": "<p>Email envoyé à {email}!</p><p>Pour continuer, cliquer sur le lien dans l'email en utilisant votre téléphone, tablette ou PC.</p><p>Aucun email? Vous ne pourrez pas créer de compte.</p>",
+  "spectating-label.label": "Observer",
+  "support.copied": "copié!",
+  "support.copy": "copier",
+  "support.details": "details",
+  "support.in-app-browser-android": "Copier coller ce lien directement dans Chrome ou Firefox",
+  "support.in-app-browser-ios": "Copier et coller ce lien directement dans Safari",
+  "support.missing-features": "Certaines fonctionnalités de votre navigateur sont manquantes.",
+  "support.supported": "supporté",
+  "support.unsupported": "non supporté",
+  "support.update-browser": "S'il vous plait, essayez de changer ou mettre à jour vers un navigateur plus récent",
+  "tips.desktop.invite": "Personne d'autre est ici. Utilisez le bouton inviter en bas à gauche pour partager cette salle.",
+  "tips.desktop.locomotion": "Utilisez les touches W A S D pour bouger. Maintenez shift pour accélérer.",
```

Comment:
```suggestion
  "tips.desktop.locomotion": "Utilisez les touches W A S D pour vous déplacer. Maintenez SHIFT pour accélérer.",
```

Reply:
How is it on you azerty keyboards?  I wonder if an issue should be made about this.

Reply:
W = Z
A = Q
S and D are the same 
Some french-speaking places (like Québec) use QWERTY keyboard